xwiki-platform-core/xwiki-platform-webjars/xwiki-platform-webjars-api/src/test/java/org/xwiki/webjars/internal/DefaultWebJarsUrlFactoryTest.java

```diff
@@ -35,25 +34,23 @@
 import org.xwiki.test.junit5.mockito.InjectMockComponents;
 import org.xwiki.test.junit5.mockito.MockComponent;
 import org.xwiki.url.ExtendedURL;
-import org.xwiki.webjars.internal.WebJarsResourceReference;
-import org.xwiki.webjars.script.WebJarsScriptService;
 import org.xwiki.wiki.descriptor.WikiDescriptorManager;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 /**
- * Unit tests for {@link org.xwiki.webjars.script.WebJarsScriptService}.
+ * Unit tests for {@link DefaultWebJarsUrlFactory}.
```

Comment:
I guess that if you had kept testing through the script service, it would not reduce the coverage. But as you prefer.

Reply:
I hesitated. But felt like a hack to artificially preserve the ratio for no good reason.
I could have added a new test on the script service to test the passing to the new component, but since they contain absolutely no logic, it seemed overkill.

Reply:
> for no good reason.

Well it's not just the ratio, it also make sure the script service does the right stuff too.